telemetry/specs/telemetry_report.mdk

```diff
@@ -424,47 +439,107 @@ The Destination Port is user configurable. The expectation is that the same
-    4 octets, including the variable optional metadata.
+* RepType (4b): Report Type
+  : Type of Telemetry report
+    * 0: INT v2.0
+    * 1: IOAM
+    * 2 – 15: Reserved
+
+* InType (4b): Inner Type
+  : Type of data embedded after Variable Optional Baseline & DS Metadata.
+    * 0: None
+    * 1: Domain Specific Extension Data
+    * 2: Ethernet
+    * 3: IPv4
+    * 4: IPv6
+    * 5-7: Reserved.
+
+* Report Length (8b): Report Length
+  : Indicates length of report header in multiple of 4-byte words, including the variable 
+    optional baseline metadata,  DS metadata and Truncated Packet or additional Domain 
+    Specific Extension Data.
+    
+* MD Length (8b): Metadata Length
+  : Indicates the length of Variable Optional Baseline Metadata and Variable Optional 
+    Domain Specific Metadata in 4-byte words.
+
+* D (1b): Dropped
+  : Indicates that at least one packet matching a drop watchlist was dropped.
 
-* NProt: Next Protocol
-  :
-    * 0: Ethernet
-    * 1: IPv4
-    * 2: IPv6
+* Q (1b): Congested Queue Association
+  : Indicates the presence of congestion on a monitored queue.
 
-* RepMdBits: Report Metadata Bits
+* F (1b): Tracked Flow Association
+  : Indicates that this telemetry report is for a tracked flow, i.e. the
+    packet matched a flow watchlist somewhere (in case of INT or IOAM) or
+    locally (in case of postcard). The report might include INT or IOAM
+    metadata beyond the inner ethernet header. Other telemetry reports are
+    likely to be received for the same tracked flow, from the same network
+    device and (in case of drop reports, postcard or path changes) from
+    other network devices.
+  
+* I (1b): Intermediate Report
+  : Indicates that a transit node sent this intermediate report for INT-MD.
+  
+* Rsvd (4b): Reserved 
+  : Should be set to zero upon transmission, and ignored upon reception
+
+* RepMdBits (16b): Report Metadata Bits
   : Bitmap that indicates which optional metadata is present in the
     telemetry report header. Each bit represents 4 octets of optional
-    metadata.
-    * bit 0 (MSB): Ingress port id (16 bits) + Egress port id (16 bits)
-    * bit 1: Hop latency
-    * bit 2: Queue ID (8 bits) + Queue occupancy (24 bits)
-    * bit 3: Egress Timestamp
-    * bit 4: Queue ID (8 bits) + Drop reason (8 bits) + Padding (16 bits)
-    * bit 5: Egress port tx utilization
+    metadata, except for bit 4,5 & 6 which represents 8 octets of optional metadata.
+    
+    * bit 0 (MSB): Reserved
+    * bit 1: Level 1 Ingress Interface ID (16 bits) & Egress Interface ID (16 bits)
+    * bit 2: Hop Latency
+    * bit 3: Queue ID (8 bits) + Queue Occupancy (24 bits)
+    * bit 4: Ingress Timestamp (64 bits)
+    * bit 5: Egress Timestamp (64 bits)
+    * bit 6: Level 2 Ingress Interface ID (32 bits) + Egress Interface ID (32 bits)
+    * bit 7: Egress Port TX Utilization
+    * bit 8: Buffer ID (8 bits) + Buffer Occupancy (24 bits)
+    * bit 9-14: Reserved. Should be set to zero.
```

Comment:
upon transmission and ignored upon reception

Reply:
Added